backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -309,6 +321,9 @@ private Bundle convertRowToFhirBundle(TestResultRow row, UUID orgId) {
                     .testkitNameId(testKitNameId)
                     .equipmentUid(equipmentUid)
                     .deviceModel(row.getEquipmentModelName().getValue())
+                    .issued(
+                        Date.from(
+                            testResultDate.atZone(zoneIdGenerator.getSystemZoneId()).toInstant()))
```

Comment:
this on the other hand uses the system default time zone to infer the time zone for text date time in the CSV (e.g. `06/14/23 12:36`, which has no time zone)
we have to do this any time there's a date time in the CSV since the format doesn't specify time zone. we should figure out a better way to determine what time zone to use